test/Bridges/Constraint/count_distinct.jl

```diff
@@ -146,6 +171,22 @@ function test_runtests_error_affine()
     return
 end
 
+function test_resolve_with_modified_not_equal_to()
```

Comment:
Could you please also add a test where there are no ranges known at all?

Reply:
See the existing `test_runtests_error_affine` above

Reply:
So this expansion still won't support that case? That's rather not useful,
and means i won't even be able to rely on this expansion doing the right thing.
That's rather unfortunate.